.gitignore

```diff
@@ -1,2 +1,4 @@
 .env
 node_modules
+dist
+.DS_*
```

Comment:
I'm always a little wary around wildcards so I consulted ChatGPT on this - it is adamant that `.DS_*` will not catch `.DS_Store` files for reasons I don't quite understand:

>  .DS_* expects there to be at least one character after .DS_, which would not match .DS_Store because there is no underscore before "Store".

> .DS*, on the other hand, matches any file starting with .DS, including .DS_Store, because the asterisk (*) wildcard does not require any characters to follow for a match.

I've gone back and forth with it trying to understand this logic but in any case, CGPT recommends explicitly including `.DS_Store` in the gitignore. If this is unnecessary, please disregard.